webrecorder/browsertrix
# Show critical errors in Crawl detail logs

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/330



### Changes

Adds new tab to crawl detail for viewing error logs

### Manual testing

1. Log in and go to Finished Crawls
2. Click a crawl -> Logs. Verify logs are shown
3. Go back to Crawls list and click through over crawl logs. Verify logs are paginated for >50 log items

### Screenshots

| Page | Image/video |
| ---- | ----------- |
| Crawl Detail - Logs |  <img width="1145" alt="Screen Shot 2023-05-02 at 2 29 43 PM" src="https://user-images.githubusercontent.com/4672952/235795864-fc5e547b-3380-41bf-bc3c-6ffeb147d6c8.png"><br /><img width="937" alt="Screen Shot 2023-05-02 at 2 29 52 PM" src="https://user-images.githubusercontent.com/4672952/235795904-d779c224-f5f5-42ba-b729-603beeb34114.png"> |
| Crawl Detail - Logs (empty) | <img width="942" alt="Screen Shot 2023-05-02 at 2 30 23 PM" src="https://user-images.githubusercontent.com/4672952/235795953-18653f7a-ae77-4824-8342-33988e50445a.png"> |
